go.mod

```diff
@@ -1,20 +1,8 @@
 module github.com/networkservicemesh/networkservicemesh
 
-require (
-	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
-	github.com/go-errors/errors v1.0.1
-	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
-	github.com/networkservicemesh/networkservicemesh/test v0.2.0
-	github.com/onsi/gomega v1.7.0
-	github.com/opentracing/opentracing-go v1.1.0
-	github.com/pkg/errors v0.8.1
-	github.com/sirupsen/logrus v1.4.2
-	github.com/uber-go/atomic v1.4.0 // indirect
-	github.com/uber/jaeger-client-go v2.17.0+incompatible
-	google.golang.org/grpc v1.23.1
-)
-
 replace (
+	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
+	github.com/census-instrumentation/opencensus-proto v0.1.0-0.20181214143942-ba49f56771b8 => github.com/census-instrumentation/opencensus-proto v0.0.3-0.20181214143942-ba49f56771b8
```

Comment:
Why are we depending on OpenCensus and thrift?

Reply:
We are depending on them indirectly through `spire`

Reply:
OK... but we haven't changed the dependencies here... we aren't pulling spire into this go.mod file... so I'm still confused...

Reply:
Yes, but we are depending on `pkg` which is depend on `spire`. And the funny thing about `go module` is that during `build` only the last replace matter. All intermediate replaces just ignored

Reply:
Ah... makes sense now.